GPflowOpt/acquisition/acquisition.py

```diff
@@ -82,14 +86,15 @@ def _optimize_models(self):
             best_idx = np.argmin([r.fun for r in runs])
             model.set_state(runs[best_idx].x)
 
-    def build_acquisition(self):
+    def build_acquisition(self, Xcand):
         raise NotImplementedError
 
     def enable_scaling(self, domain):
         """
         Enables and configures the :class:`.DataScaler` objects wrapping the GP models.
+        
         :param domain: :class:`.Domain` object, the input transform of the data scalers is configured as a transform
-         from domain to the unit cube with the same dimensionality.
+                       from domain to the unit cube with the same dimensionality.
```

Comment:
seems multiline param explanation should be properly indented for sphinx to display it correctly. Perhaps using the python line continuation \ works too